code/modules/mob/living/basic/vermin/cockroach.dm

```diff
@@ -36,7 +36,7 @@
 	AddComponent(/datum/component/squashable, squash_chance = 50, squash_damage = 1)
 
 /mob/living/basic/cockroach/death(gibbed)
-	if(SSticker.mode.station_was_nuked) //If the nuke is going off, then cockroaches are invincible. Keeps the nuke from killing them, cause cockroaches are immune to nukes.
+	if(!null && SSticker.mode.station_was_nuked) //If the nuke is going off, then cockroaches are invincible. Keeps the nuke from killing them, cause cockroaches are immune to nukes.
```

Comment:
This doesn't work, use `?.` if you want to check for a null here.

Reply:
This runtime also isn't caused by there being no database in a dev enviroment as you claim in the PR descrition none of this has anything to do with the DB